modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java

```diff
@@ -289,6 +289,12 @@ private Map<String, String> _getAllParametersMap(
 			allParametersMap.put(key, String.valueOf(entry.getValue()));
 		}
 
+		if (!Objects.isNull(ddmDataProviderRequest.getLocale())) {
+			allParametersMap.put(
+				"ddm.locale",
+				String.valueOf(ddmDataProviderRequest.getLocale()));
+		}
+
```

Comment:
@p-albuquerque why did you decide to use `Objects.isNull`? can we simplify to:

```
Locale locale = ddmDataProviderRequest.getLocale();

if (locale != null) {
	allParametersMap.put("ddm.locale", LocaleUtil.toLanguageId(locale));
}
```
?

Reply:
In addition, I think we should use the `LocaleUtil.toLanguageId` since we are doing `LocaleUtil.fromLanguageId` int `JSONWebServiceServlet` class

Reply:
Copy that! thanks